components/postcode-lookup/defaults.js

```diff
@@ -0,0 +1,31 @@
+/* eslint max-len: 0, no-process-env: 0 */
+
```

Comment:
either the naming of the file or the exported object structure can be improved. It looks like the file is just default error messages, so you can either rename the file to something like defaultErrorMessages.js or change the structure of the object returned to something like:

`module.exports = {
    errorMessages: {
        CANT_FIND: '...',
        ...;
    }

}`

Reply:
I've now moved the variables linked to an issue with the postcode lookup to within the `POSTCODE_ERROR` structure within this file. The rest of the properties are not issues with the lookup, so have kept them as independent variables.